rest-server/src/main/java/io/github/nubesgen/configuration/NubesgenConfiguration.java

```diff
@@ -228,13 +242,20 @@ public boolean isRuntimeMaven() {
             RuntimeType.SPRING.equals(this.getRuntimeType()) ||
             RuntimeType.QUARKUS.equals(this.getRuntimeType()) ||
             RuntimeType.QUARKUS_NATIVE.equals(this.getRuntimeType()) ||
+            RuntimeType.MICRONAUT.equals(this.getRuntimeType()) ||
+            RuntimeType.DOCKER_MICRONAUT.equals(this.getRuntimeType()) ||
             RuntimeType.JAVA.equals(this.getRuntimeType())
         );
     }
 
     @JsonIgnore
     public boolean isRuntimeGradle() {
-        return (RuntimeType.SPRING_GRADLE.equals(this.getRuntimeType()) || RuntimeType.JAVA_GRADLE.equals(this.getRuntimeType()));
+        return (
```

Comment:
should not this be better as a method of `RuntimeType`. e.g. `RuntimeType::isGradle?`